pkg/controller/job/job_controller.go

```diff
@@ -753,6 +763,26 @@ func (jm *Controller) syncJob(ctx context.Context, key string) (forget bool, rEr
 		job.Status.StartTime = &now
 	}
 
+	jm.mutex.Lock()
```

Comment:
This block of code should be in a utility method.

Reply:
probably in backoff_utils

Reply:
Done